pom.xml

```diff
@@ -112,11 +112,13 @@
           <url>https://oss.sonatype.org/content/groups/public</url>
         </repository>
         <!-- for j3d core, utils and vector  -->
+        <!--
```

Comment:
if we don't need it.. just remove it.. any reason to keep it around & commented out?

Reply:
Notes say for j3d core utils and vector ...  vector lib (as well as tools lib) has always been a pain, Oracle doesn't make it available in any public repo and neither have become fully open source ... the tuft repo had it ...
The error from tufts was a PIX TLS chain is broken, we still need the libs.  I commented out - if any of the nodes purge their .m2 we'll quickly find all the missing ones we probably need to put into mrl's repo

Reply:
In short - dunno what happened with tufts - but we certainly need vector.lib and others - so i commented it out for a reference.